index.html

```diff
@@ -206,8 +206,12 @@
         <dfn>PaymentRequest</dfn> interface
       </h2>
       <pre class="idl">
-        [Constructor(sequence&lt;PaymentMethodData&gt; methodData, PaymentDetails details, optional PaymentOptions options),
-        SecureContext]
+        [Constructor(
+          optional DOMString paymentRequestID, 
```

Comment:
I'm not a fan of putting it in PaymentDetails or PaymentOptions because it relegates the field to be second class.